src/adapters/utils/markdown-to-plaintext.ts

```diff
@@ -6,13 +6,16 @@ import plainTextPlugin from "markdown-it-plain-text";
  * @param markdown
  * @returns
  */
-export function markdownToPlainText(markdown: string | null): string | null {
+export function markdownToPlainText(markdown?: string | null): string {
   if (!markdown) {
-    return markdown;
+    return "";
```

Comment:
Is this right?

Reply:
I guess empty strings are avoided so no it's not